[Durable Objects] Update _index.md (#10214)
cloudflare/cloudflare-docs: content/durable-objects/_index.md

{{<description>}}

> A globally distributed coordination API with strongly consistent storage.

Author: Can I ask why this is this strict (don't say style guide please) - ? 

Durable Objects is a primitive. If say "Build applications using a globally distributed [...]" it reads worse.

> A globally distributed coordination API with strongly consistent storage.

Reviewer: Let me check and get back to you, because I initially created a description similar to yours but I was asked to change it.

> A globally distributed coordination API with strongly consistent storage.

Reviewer: In the end, no, it doesn't have to.

However, an alternative could be:

**_Build globally, distributed and coordinated APIs with consistent storage._**

> A globally distributed coordination API with strongly consistent storage.

Author: We're the API - we absolve them of having to build all of that ;-)

{{</description>}}

{{<plan type="paid">}}

> Durable Objects provide a powerful API for coordinating the actions of multiple clients or users, each with private, transactional and strongly consistent storage attached.

Author: Neither. Actions in a truly general sense. Messages. Cursor events. Character/player positions. We should not be too precise here — this is the summary. Needs to give people inspiration, not cause them to disqualify themselves.

> Developers use Durable Objects to to build collaborative editing, interactive chat, multiplayer games and other applications that have a strong need to coordinate the actions of multiple clients, without requring you to build serialization and coordination primitives on your own. 

Reviewer: ```suggestion
Use Durable Objects to build collaborative editing tools, interactive chat, multiplayer games and applications that need coordination among multiple clients, without requiring you to build serialization and coordination primitives on your own. 
```

> You can create millions of (or more) Durable Objects, representing a document, chat session, multiplayer game or other aspect of your system where events need to be coordinated or serialized. Each Durable Object is automatically located as close to the client or user as possible, minimizing latency between the client and Durable Objects.

Reviewer: ```suggestion
```

Reviewer: I wouldn't add this information here. 

My suggestion is to create a separate page (for example: "Durable Objects Architecture") where we dive deeper into how Durable Objects actually allow a user to minimize latency, achieve coordination, etc. 

At the moment, it's still quite unclear how Durable Objects communicate with one another.

---

## Features

{{<feature header="Transactional Storage API" href="/durable-objects/learning/in-memory-state/">}}

Understand how Durable Objects can act as a powerful coordination API for managing the actions of multiple clients or events.

{{</feature>}}

{{<feature header="Transactional Storage API" href="/durable-objects/api/transactional-storage-api/">}}

> Learn how Durable Objects provide strongly consistent, serializable key-value storage.

Reviewer: ```suggestion
Learn how Durable Objects provide consistent, serializable key-value storage.
```

Author: `strongly consistent` has real meaning to developers here

{{</feature>}}

{{<feature header="Hibernatable WebSockets API" href="/durable-objects/api/hibernatable-websockets-api/">}}

Learn how the Hibernatable WebSockets API allows you to manage the connections of multiple clients at scale.

{{</feature>}}

{{<feature header="Durable Objects Alarms" href="/durable-objects/api/alarms-in-durable-objects/">}}

> Learn how to trigger a Durable Object to wake up and perform compute in the future at customizable intervals.

Reviewer: ```suggestion
Learn how to use alarms to trigger a Durable Object and perform compute in the future at customizable intervals.
```

## More resources

{{<resource-group>}}

> {{<resource header="Built with Durable Objects" href="https://workers.cloudflare.com/built-with/collections/durable-objects/" icon="reference-architecture">}}See what other developers have built with Durable Objects.{{</resource>}}

Reviewer: ```suggestion
{{<resource header="Built with Durable Objects" href="https://workers.cloudflare.com/built-with/collections/durable-objects/" icon="reference-architecture">}}Browse what other developers are building with Durable Objects.{{</resource>}}
```
